### DISCUSSION TITLE
Using duckdb to speedup lookup

### DISCUSSION THREAD

**Discussion by WolfgangFahl** [SmartDataAnalytics/OpenResearch#118] [Ideas]
A few weeks ago in december I learned about duckdb during our doctoral
seminar at RWTH Aachen i5 where i am a research assistant 3/4 of my time.

http://dbis.rwth-aachen.de/cms/staff/till-doehmen Till Döhmen talked
about duckdb and later recommended it for our ConfIDent project

See http://confident.bitplan.com/

I'd love to integrate Duckdb with the
https://pypi.org/project/pylodstorage/ for which i am a maintainer and which we use in our current demos and protoypes in OpenResearch / ConfIDent

**Comment by tdoehmen** [comment]
@WolfgangFahl I'm happy to help, let me know if you have any questions
